cli/foundation-models/system/finetune/text-generation/text-generation-pipeline.yml

```diff
@@ -0,0 +1,87 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+
+experiment_name: text-generation-samsum
+
+inputs:
+  compute_model_import: gpu-cluster-big
+  compute_preprocess: gpu-cluster-big
+  compute_finetune: gpu-cluster-big
+  compute_model_evaluation: gpu-cluster-big
+
+  # specify the foundation model available in the azureml system registry
+  mlflow_model_path: 
+    path: azureml://registries/azureml/models/Llama-2-7b/versions/5
```

Comment:
Is it not possible to pass the latest version here similar to picking the latest pipeline component? azureml://registries/azureml/components/text_generation_pipeline/labels/latest

Reply:
no, tested it, its still not possible to give latest label